### PR TITLE
CFD-411: Voices & views carousel should link to Article detail page

### PR DESCRIPTION
**Pull request by solker**
#567

### CODE REVIEW

File: capacity4more/modules/c4m/features/c4m_features_homepage/templates/voices-views-carousel.tpl.php [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
         <div ng-style="{'background-image': 'url(' + slide.image + ')'}"  class="bgimage">
-          <div class="intro-text">
-            <span class="title">
-            {{ slide.title }}
-          </span>
-          <span class="text">
-            {{ slide.text }}
-          </span>
-          </div>
+          <a href="/articles">
```
Comment (solker) [inline capacity4more/modules/c4m/features/c4m_features_homepage/templates/voices-views-carousel.tpl.php]: Is good way to make link to articles page?

Comment (nader77) [inline capacity4more/modules/c4m/features/c4m_features_homepage/templates/voices-views-carousel.tpl.php]: No, use ``<?php print url('articles') ?>`` Disable pURL if you have to by sending the options to the template from the plugin.

**Comment by zero2one** [comment]
@solker Needs a re-roll

**Comment by solker** [comment]
@zero2one re-rolled and implemented feature from comment from Maarten on Jira. 

>Ilya Manyahin, can you also add a link on "Voices & views"?
"clicking "Voices & views" leads to voices & views overview page."

**Comment by zero2one** [comment]
@solker The build is failing due to a broken test…

**Comment by MPParsley** [comment]
Fixed behat test with commit 0d3bd5aeda12be8dcfaa614bf8a5214e31c0b7fb - https://github.com/capacity4dev/capacity4more/pull/630

FYI: there can be different items in the carousel so to avoid random results we test if the id is present instead of a title.

**Comment by nader77** [comment]
@solker This is ready ?

**Comment by solker** [comment]
@nader77 yes.

**Comment by nader77** [comment]
@solker Add a screenshot and assign to me.

**Comment by solker** [comment]
@nader77 

9058f3f0-d2ce-11e4-88dc-aa3c070d847e

**Comment by nader77** [comment]
PR is ready

**Pull request merged**